src/main/webapp/app/fileupload/overview/file-upload-submission/file-upload-submission.component.html

```
@@ -1,4 +1,4 @@
-@if (displayHeader() && fileUploadExercise() && participation()) {
+@if (displayHeader() && examMode() && fileUploadExercise() && participation()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)